docs/howto/define_extra_link.rst

```diff
@@ -56,3 +56,75 @@ The following code shows how to add extra links to an operator:
 You can also add a global operator extra link that will be available to
 all the operators through airflow plugin. Learn more about it in the
 :ref:`plugin example <plugin-example>`.
+
+
+Add Operator Link to Existing Operators
+---------------------------------------
+
+You can also add (or override) an extra link to an existing operators
+through Airflow plugin.
+
+For example, the following Airflow plugin will add an Operator Link on all
+tasks using :class:`~airflow.operators.gcs_to_s3.GoogleCloudStorageToS3Operator` operator.
+
+**Adding Operator Links to Existing Operators**
+``plugins/extra_link.py``:
+
+.. code-block:: python
+
+  from airflow.plugins_manager import AirflowPlugin
+  from airflow.models.baseoperator import BaseOperatorLink
+  from airflow.operators.gcs_to_s3 import GoogleCloudStorageToS3Operator
+
+  class S3LogLink(BaseOperatorLink):
+      name = 'S3'
+
+      # Add list of all the operators to which you want to add this OperatorLinks
+      # Example: operators = [GoogleCloudStorageToS3Operator, GoogleCloudStorageToBigQueryOperator]
+      operators = [GoogleCloudStorageToS3Operator]
+
+      def get_link(self, operator, dttm):
+          return 'https://s3.amazonaws.com/airflow-logs/{dag_id}/{task_id}/{execution_date}'.format(
+              dag_id=operator.dag_id,
+              task_id=operator.task_id,
+              execution_date=dttm,
+          )
+
+  # Defining the plugin class
+  class AirflowExtraLinkPlugin(AirflowPlugin):
+      name = "extra_link_plugin"
+      operator_extra_links = [S3LogLink(), ]
+
+
+
+**Overriding Operator Links of Existing Operators**:
+
+For :class:`~airflow.gcp.operators.bigquery.BigQueryOperator` you can create
```

Comment:
This needs re-wording slightly, as one reading might take this to mean it only applies to this one class.

Reply:
```
It is also possible to replace a built in link on an operator via a Plugin. For example 
:class:`~airflow.gcp.operators.bigquery.BigQueryOperator` includes a link to the GCP
console, but if we wanted to change that link we could:

Reply:
Updated